user-interface/src/main/java/life/qbic/datamanager/views/projects/create/AddProjectDialog.java

```diff
@@ -80,8 +82,11 @@ public AddProjectDialog(FinanceService financeService,
     this.experimentalInformationLayout = new ExperimentalInformationLayout(
         ontologyLookupService);
 
-    List<Contact> knownContacts = contactRepository.findAll().stream()
-        .map(contact -> new Contact(contact.fullName(), contact.emailAddress())).toList();
+    List<Contact> knownContacts = new ArrayList<>();
+    try {
+      knownContacts = contactRepository.findAll().stream()
+          .map(contact -> new Contact(contact.fullName(), contact.emailAddress())).toList();
+    } catch (AccessDeniedException e) {}
```

Comment:
I fear this can not be changed to return a result object, which is then handled dependent on the viewers permissions? Since this is not an exceptional case, having an exception with try catch block seems dangerous